### PR TITLE
A0-2741: Fix network dropdown ignored in account import flow

### PR DESCRIPTION
**Pull request by youPickItUp**

### CODE REVIEW

File: packages/extension-ui/src/Popup/ImportSeed/index.tsx [hunk 1]
```diff
@@ -64,7 +63,18 @@ function ImportSeed(): React.ReactElement {
...
   const _onPreviousStep = useCallback(() => setStep((step) => step - 1), []);
 
-  const _onChangeNetwork = useCallback((newGenesisHash: string) => setGenesis(newGenesisHash), []);
+  const _onChangeNetwork = useCallback(
+    (newGenesisHash: string) => {
+      if (!account) {
+        return;
+      }
+
+      setAccount({ ...account, genesis: newGenesisHash });
```
Comment (jalooc) [inline packages/extension-ui/src/Popup/ImportSeed/index.tsx:72]: Maybe simpler just to
```suggestion
      if (account) {
        setAccount({ ...account, genesis: newGenesisHash });
      }
```

**Review by jalooc — APPROVED** [review]
Huh, seems that before this fix, `account.genesis` was set based on `genesis` that was set... one step after `account.genesis` was set? 😶

File: packages/extension-ui/src/Popup/ImportSeed/index.tsx (hunk 1 above)
Comment (youPickItUp) [reply packages/extension-ui/src/Popup/ImportSeed/index.tsx:72]: I'd leave it as it is, if you don't mind :pray:

Comment (jalooc) [reply packages/extension-ui/src/Popup/ImportSeed/index.tsx:72]: Sure 👍

**Pull request merged**